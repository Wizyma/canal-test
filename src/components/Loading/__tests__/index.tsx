import { render } from '@testing-library/react';

import Component from '..'

test(`It should render the ${Component.name} component`, () => {
  const { asFragment } = render(<Component />)

  expect(asFragment()).toMatchSnapshot()
})