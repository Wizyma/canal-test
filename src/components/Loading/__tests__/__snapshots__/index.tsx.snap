// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the Loading component 1`] = `
<DocumentFragment>
  <div
    class="jsx-2070862285 item justify sm jsx-4216129763 "
  >
    <div
      class="jsx-16278101 card "
    >
      <div
        class="jsx-2956695818 content "
      >
        <div
          class="jsx-3255440017 loading-container"
        >
          <span
            class="jsx-3255440017 loading"
          >
            <i
              class="jsx-3255440017"
            />
            <i
              class="jsx-3255440017"
            />
            <i
              class="jsx-3255440017"
            />
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
